i18next-parser.config.js

```diff
@@ -0,0 +1,112 @@
+// i18next-parser.config.js
+
+export default {
```

Comment:
I think it would be simpler to replace this file by just what we are using and let the defaults not mentioned. It is also safer to do so because you get less breaking changes over time if the package maintainers introduce breaking changes on config properties @rperon